flang/module/cudadevice.f90

```diff
@@ -0,0 +1,21 @@
+!===-- module/cudedevice.f90 -----------------------------------------------===!
```

Comment:
Why can't these USE renames be in the existing `flang/module/__cuda_builtins.f90`?  Is it because the program is allowed to also explicitly USE `cudadevice`?

Reply:
Yeah that was the idea. The `cudadevice` module can be explicitly USE'd for example to have access to these procedures inside a CUF Kernel do.